src/NRedisStack/Bloom/BloomCommandBuilder.cs

```diff
@@ -0,0 +1,95 @@
+using NRedisStack.Literals;
+using NRedisStack.RedisStackCommands;
+using StackExchange.Redis;
+namespace NRedisStack
+{
+
+    public static class BloomCommandBuilder
```

Comment:
IMHO comment the class too. Let's explain to users why this exists, why it's an issue.